run/django/noxfile_config.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default TEST_CONFIG_OVERRIDE for python repos.
+
+# You can copy this file into your directory, then it will be inported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    "ignored_versions": ["2.7", "3.6", "3.7"],
+
```

Comment:
```suggestion
# Old samples are opted out of enforcing Python type hints
# All new samples should feature the
'enforce_type_hints': True,
```

Reply:
We had previously asked that types be documented as part of samples, but were not consistent about enforcing it. We're now slowly rolling it into new samples via this config in the noxfile_config - hopefully it should not be brainbreaking work to change but if it's proving to be a nightmare, plz ping me because it was my idea and if it's awful it means I definitely didn't consider whatever factor is making it awful 😄

Reply:
Turning this config on will add a check to lint and that check will show you where changes need to be made. Unfortunately they don't add automatically, but at least there's a checker!

Reply:
I've added typehinting for my new code, but the check is still failing because it's seeing errors in noxfile.py. I'm not sure *which* noxfile it is, but the fact that there's a big "do not edit this file ever' warning makes me think I shouldn't try and fix that.

Reply:
This PR now adds typechecking to `noxfile-template.py`.